src/resmom_win/start_exec.c

```diff
@@ -3744,6 +3936,24 @@ job_nodes_inner(struct job *pjob, hnodent **mynp)
 					phv[hp->hn_vlnum].hv_mem   = ndmem;
 					hp->hn_vlnum++;
 				}
+
+				vmp = &pjob->ji_assn_vnodes[assn_index];
+				vmp->vn_node = assn_index++;
+				if (hp != NULL)
+					vmp->vn_host = hp;
+				if (pmm != NULL)
+					vmp->vn_vname = strdup(pmm->mvm_name);
+				if (vmp->vn_vname == NULL) {
+					if (vmp->vn_hname != NULL) {
+						free(vmp->vn_hname);
+						vmp->vn_hname = NULL;
+					}
+					return (PBSE_SYSTEM);
+				}
+				vmp->vn_cpus  = vnncpus;
```

Comment:
Are there two spaces before the =?

Reply:
There sure was and the others. Removed the extra space.